mindjunkies/courses/migrations/0005_course_tags.py

```
@@ -0,0 +1,20 @@
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('courses', '0004_coursetoken_intro_video_alter_coursetoken_motivation'),
+        ('taggit', '0006_rename_taggeditem_content_type_object_id_taggit_tagg_content_8fc721_idx'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='course',
+            name='tags',
+            field=taggit.managers.TaggableManager(blank=True, help_text='A comma-separated list of tags.', through='taggit.TaggedItem', to='taggit.Tag', verbose_name='Tags'),
```

Ruff (0.8.2)

Line too long (174 > 119)

(E501)